src/civic/index.js

```diff
@@ -118,15 +118,28 @@ const translateRelevance = (evidenceType, evidenceDirection, clinicalSignificanc
     switch (evidenceType) { // eslint-disable-line default-case
         case 'Predictive': {
             switch (clinicalSignificance) { // eslint-disable-line default-case
-                case 'Sensitivity':
+                case 'Sensitivity': {
+                    switch (evidenceDirection) { // eslint-disable-line default-case
+                        case 'Does Not Support': { return 'no response'; }
+
+                        case 'Supports': { return clinicalSignificance.toLowerCase(); }
+                    }
+                }
                 case 'Adverse Response':
+
                 case 'Reduced Sensitivity':
 
                 case 'Resistance': {
```

Comment:
now that we are allowing the evidenceDirection to include "Does not support" we have to assume it can happen for any of the inputs. Currently this would not account for "Does not support" "Resistance". Since there is only 1 case we currently want to capture for "Does not support" it would probably make more sense to have this at the top-level so that any we haven't defined just all throw throw the "unable to process" error. Does that make sense? Something like

```
if (evidenceDirection == 'Does not support') {
  // new switch that only covers the cases we care about
} else if (evidenceDirection == "Supports") {
  // the original switch statement
}

throw Error(....)
```

Reply:
I've re-ordered this so that the check for evidenceDirection is at the top.
I'm not sure if the 'switch' blocks within the if/else statement need a 'break' after them.

Reply:
Made a comment, having the break will do the same thing effectively b/c it will jump to the error thrown at the end of the function. it is less redundant to break vs re-throw the error though